### DISCUSSION TITLE
最新的1.6.0版本在railway部署出现问题

### DISCUSSION THREAD

**Discussion by l-deo** [zdz/ServerStatus-Rust#116] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/24577548/205717591-56d2db77-8d85-4ff3-a8d6-81a8cbef51ec.png)
在railway部署以后，始终处于转圈状态。

**Comment by zdz** [comment, marked as answer]
是新界面的ui逻辑，当前没有主机就会一直转圈，等待有主机上报，随便一个主机正常上报上就可以了，正常现象，~~后面我调整一下~~，感觉不用调整
![image](https://user-images.githubusercontent.com/152173/205794476-2b71d762-db6c-4ffe-8656-f775fe87e166.png)
